resources/assets/js/volumes/components/filterListComponent.vue

```diff
@@ -2,7 +2,7 @@
     <span>
         <strong>with<span v-if="rule.negate">out</span></strong>
         <span v-text="name"></span>
-        <strong v-if="dataName" v-text="dataName"></strong>
+        <strong v-if="dataSpecs" v-text="dataSpecs"></strong>
```

Comment:
IMO the image count should not be bold, too. Instead it could even be muted:
```suggestion
        <strong v-if="dataName" v-text="dataName"></strong>
        <span class="text-muted v-text="fileCount"></span>
```